### DISCUSSION TITLE
GraphQL mutations

### DISCUSSION THREAD

**Discussion by sjennolkaya** [Giorgi/GraphQLinq#18] [General]
Hi Giorgi, I like very much your GraphQLinq package. Do you have plans to implement graphql mutations soon?

**Comment by Giorgi** [comment]
Hi,

Yes, I would like to implement mutations, but unfortunately, I don't have much free time to work on the library.

**Comment by nerdynomadd** [comment]
HI ! 

I've been working on a lib to make GraphQL calls using the Linq syntax, basically it was first a fork from your lib but ended to have it's own engine with support for both queries and mutations. 
While this features is not available on Giorgi's lib you can take a look at mine, I'm sure it can help you cover at least a large part of your GraphQL needs.

Here's the link: https://github.com/Kakktuss/LinqToGraphQL
